src/router/index.ts

```diff
@@ -29,6 +37,9 @@ const routes = [
             title: 'View All Activities' + suffix,
             metaTags: defaultMetaTags,
         },
+        beforeEnter() {
+            window.location.href = cChainURL
```

Comment:
We should probaby wrap this into into an if statement to check whether we are already on the `cChainURL`. To avoid infinite loops in case of a deployment rollback and during deployment.